api/src/controllers/user.js

```diff
@@ -724,9 +724,15 @@ router.put(
 
     const tx = await User.sequelize.transaction();
     if (team && Array.isArray(team)) {
+      const existingTeams = await Team.findAll({
+        where: {
+          organisation: user.organisation,
+          _id: { [Op.in]: team },
+        },
+      });
```

Comment:
J'ai un doute sur le fait de mettre ça dans la transaction aussi : 

```suggestion
      const existingTeams = await Team.findAll({
        where: {
          organisation: user.organisation,
          _id: { [Op.in]: team },
        },
        transaction: tx
      });
```